src/biogeophys/IrrigationMod.F90

```diff
@@ -583,6 +584,12 @@ subroutine CheckNamelistValidity(this, use_aquifer_layer)
                errMsg(sourcefile, __LINE__))
     end if
 
+    if (use_aquifer_layer .and. use_hillslope) then
+          write(iulog,*) ' ERROR: use_hillslope and use_aquifer_layer may not be used simultaneously'
+          call endrun(msg=' ERROR: use_hillslope and use_aquifer_layer cannot both be set to true' // &
+               errMsg(sourcefile, __LINE__))
+    end if
```

Comment:
- [ ] [OPTIONAL] I appreciate your adding this consistency check, but is there a better place to do it than here, since I don't think it's related to irrigation? (e.g., in HillslopeHydrologyMod's init method?)